setup.py

```diff
@@ -68,9 +79,7 @@ def run(self):
     author='HeikoHeiko',
     author_email='heiko@brainbot.com',
     url='https://github.com/raiden-network/raiden',
-    packages=find_packages(
-        exclude=["raiden.tests", "raiden.tests.*"]
-    ),
+    packages=find_packages(),
```

Comment:
We include also the packages needed for testing. Testing dependencies should in principle imo not be added in a production release.

For the purpose of the smoke test though I can see why we need this one test but wonder if we could somehow split the dependencies. Does smoke test depend on everything our normal tests do or could a more minimal dependency be extracted?